units/crons.go

```diff
@@ -250,14 +250,17 @@ func hostTerminationJobs(ctx context.Context, env evergreen.Environment, _ time.
 		"impact":    "hosts termination interrupted",
 	}))
 	catcher.Wrap(err, "finding hosts spawned by tasks to terminate")
-
 	for _, h := range hosts {
 		jobs = append(jobs, NewHostTerminationJob(env, &h, HostTerminationOptions{
 			TerminateIfBusy:   true,
 			TerminationReason: "host spawned by task has gone out of scope",
 		}))
 	}
-
+	grip.Info(message.Fields{
```

Comment:
This might not be that accurate since the jobs can come from other sources as well. I thought we could just use [the existing Amboy stats collector](https://github.com/evergreen-ci/evergreen/blob/31a02a033fc94e0321525501b398062ce5c33315/units/stats_amboy.go#L68-L93) and log the termination queue's stats there (where `Stats(ctx).Pending` would tell you the number of jobs in the host termination queue.

Anyways, if you want to move it there and/or just use the `q.Stats` method, that might be easier/cleaner, but I don't have strong opinions.

Reply:
I figured this is still the spot that has most of the jobs, but the stats collector does seem legit, I can try that

Reply:
@Kimchelly I added this for all queue groups to keep it general and if this will eventually be helpful -- lmk if this seems like too much logging / overkill. I can test this in staging as well.